common/datasets/switchboard/lexicon.py

```diff
@@ -0,0 +1,147 @@
+"""
+Functions related to lexicon creation
+"""
+__all__ = ["get_special_lemma_lexicon", "get_text_lexicon", "get_bliss_lexicon"]
+from sisyphus import tk
+
+import os
+
+from i6_core.datasets.switchboard import (
+    DownloadSwitchboardTranscriptionAndDictJob,
+    CreateSwitchboardLexiconTextFileJob,
+)
+from i6_core.lexicon import LexiconFromTextFileJob
+from i6_core.lexicon.modification import WriteLexiconJob, MergeLexiconJob
+from i6_core.lib import lexicon
+
+from .constants import SUBDIR_PREFIX
+
+
+def get_special_lemma_lexicon() -> lexicon.Lexicon:
+    """
+    Generate the special phonemes/lemmas for Switchboard
+    """
+    lex = lexicon.Lexicon()
+
+    tags = ["[SILENCE]", "[NOISE]", "[VOCALIZED-NOISE]", "[LAUGHTER]"]
+    tag_to_phon = {
+        "[SILENCE]": "[SILENCE]",
+        "[NOISE]": "[NOISE]",
+        "[VOCALIZED-NOISE]": "[VOCALIZEDNOISE]",
+        "[LAUGHTER]": "[LAUGHTER]",
+    }
+    for tag in tags:
+        lex.add_phoneme(tag_to_phon[tag], variation="none")
+
+    # add non-special lemmas
+    for tag in tags[1:]:  # silence is considered below
+        lex.add_lemma(
+            lexicon.Lemma(
+                orth=[tag],
+                phon=[tag_to_phon[tag]],
+                synt=[],
+                eval=[[]],
+            )
+        )
+
+    # create special lemmas
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[SENTENCE-END]"], synt=["</s>"], special="sentence-boundary"
+        )
+    )
+
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[sentence-begin]"],
+            synt=["<s>"],
+            eval=[[]],
+            special="sentence-begin",
+        )
+    )
+
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[sentence-end]"], synt=["</s>"], eval=[[]], special="sentence-end"
+        )
+    )
+
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[SILENCE]", ""],
+            phon=["[SILENCE]"],
+            synt=[],
+            eval=[[]],
+            special="silence",
+        )
+    )
+
+    lex.add_lemma(
+        lexicon.Lemma(orth=["[UNKNOWN]"], synt=["<unk>"], eval=[[]], special="unknown")
+    )
+
+    return lex
+
+
+def get_text_lexicon(subdir_prefix: str = SUBDIR_PREFIX):
+    """
+    :param subdir_prefix:
```

Comment:
```suggestion
    :param subdir_prefix: alias prefix name
```

Reply:
I think this is self-explainatory, and in some cases this is also output and not only alias.

Reply:
Yes it mostly is. Was just wondering because for some function this is the docstring and then there were docs where this was just empty, so i copied it over.